app/models/application.rb

```diff
@@ -9,6 +9,20 @@ def self.current_application_year
     Time.now.year
   end
 
+	def getAmountPaid
```

Comment:
Is it possible to do something simplier such as making amount_paid a default value in the database schema so you dont need to check for nil amnt?